packages/syft/src/syft/__init__.py

```diff
@@ -159,3 +161,100 @@ def _orchestra() -> Orchestra:
 
 def search(name: str) -> SearchResults:
     return Search(_domains()).search(name=name)
+
+
+def launch(
+    # node information and deployment
+    name: str | None = None,
+    node_type: str | NodeType | None = None,
+    deploy_to: str | None = None,
+    node_side_type: str | None = None,
+    # worker related inputs
+    port: int | str | None = "auto",
+    processes: int = 1,  # temporary work around for jax in subprocess
+    local_db: bool = False,
+    dev_mode: bool = False,
+    cmd: bool = False,
+    reset: bool = False,
+    tail: bool = False,
+    host: str | None = "0.0.0.0",  # nosec
+    tag: str | None = "latest",
+    verbose: bool = False,
+    render: bool = False,
+    enable_warnings: bool = False,
+    n_consumers: int = 0,
+    thread_workers: bool = False,
+    create_producer: bool = False,
+    queue_port: int | None = None,
+    in_memory_workers: bool = True,
+    skip_signup: bool = False,
+    default_admin: bool = False,
+) -> DomainClient:
+    node = Orchestra.launch(
+        name=name,
+        node_type=node_type,
+        deploy_to=deploy_to,
+        node_side_type=node_side_type,
+        port=port,
+        processes=processes,
+        local_db=local_db,
+        dev_mode=dev_mode,
+        cmd=cmd,
+        reset=reset,
+        tail=tail,
+        host=host,
+        tag=tag,
+        verbose=verbose,
+        render=render,
+        enable_warnings=enable_warnings,
+        n_consumers=n_consumers,
+        thread_workers=thread_workers,
+        create_producer=create_producer,
+        queue_port=queue_port,
+        in_memory_workers=in_memory_workers,
+    )
+
+    client = node.login(
+        email="info@openmined.org",
+        password="changethis",  # nosec
+        verbose=False,
+        suppress_warnings=not default_admin,
+    )
+
+    # so that the user doesn't need to keep up with a node_handlne and client
+    client.land = node.land
+
+    if not skip_signup:
+        if not default_admin:
+            print("\nConfiguring admin account...")
+            name = input("\tAdmin Name:")
+            email = input("\tAdmin Email:")
+            password = getpass("\tAdmin Password:")
+
+            client.me.set_name(name)
+            client.me.set_email(email)
+            client.me.set_password(password)
+
+        # set default email notifier (note: each email account can only send 300 emails a day,
+        # and it might go to people's junk folders.)
+        emails_and_passwords = list()  # noqa: C408
+        emails_and_passwords.append(
```

Comment:
this is part of syft.launch now, does not sound like a good idea to hardcode things here. If we really want this can we add environment variables for this?

Reply:
Hmm agreed... Since hagrid will be deprecated soon, we anyway need to refactor `sy.orchestra.launch` and give it a proper home within syft. Additionally, this gives us the opportunity to implement a bootstrapping mechanism that @madhavajay was asking.

I'm thinking on the lines of a `bootstrap` module (a more generic variant of `demo.py`) that allows us to setup arbitrary/static data in the node. This entails a couple of questions:

- Currently `.demo()` is being called by the client, but who should really bootstrap the data? Syft backend on node start as per args?  Client-DS? Client-DO? And how do we control what data gets fed into the backend?
- How does this operate in different modes of sy.launch, namely - inmemory, process, single_container, k8s. Because in k8s you can only "connect" to a node and not "launch" it, so we should expect some baseline information to be already present.

Reply:
Yes, I think we should split this up into two PRs:
- small textual changes / flags which we want to change
- adding some kind of demo mode, which to me might be better handled by the planned work to add syft demo data fixutres into the code base that let you easily run "scenarios" without having to load up a notebook and copy and paste to get to a known good state

Reply:
> "can we add environment variables for this"

Down to split this into two PRs — but I strongly disagree with environment variables as a solution to beginner-friendly automatic configuration.

In an intro tutorial, a beginner should be able to not know how to use the command line (as in... have never heard of the command line, environment variables, etc.). Beginners are statisticians, not software engineers.